tests/ground_destroy_test.cpp

```diff
@@ -25,7 +25,7 @@ TEST_CASE( "pavement_destroy", "[.]" )
     const ter_id flat_roof_id = ter_id( "t_flat_roof" );
     REQUIRE( flat_roof_id != t_null );
 
-    const bool zlevels_set = get_option<bool>( "ZLEVELS" );
+    const bool zlevels_set = true;
     INFO( "ZLEVELS is " << zlevels_set );
```

Comment:
```suggestion
```
Might as well just delete this entirely.  No point having a constant value be output.

(Similarly for the other changes in this file).